svelte-development/src/helper.js

```diff
@@ -20,3 +21,314 @@ export function findRegionColor(region) {
+
+    //A09
+    if (this.questions.get("Q23") && this.questions.get("Q04") && this.questions.get("Q6")) {
+      if(this.modes.get("A09") !== undefined) {
+        this.modes.set("A09", true);
+      }
+    }
+
+    //A10
+    if (this.questions.get("Q03") && this.questions.get("Q05")) {
+      if(this.modes.get("A10") !== undefined) {
+        this.modes.set("A10", true);
+      }
+    }
+
+    //A11
+    if (this.questions.get("Q17") && this.questions.get("Q18")) {
+      if(this.modes.get("A11") !== undefined) {
+        this.modes.set("A11", true);
+      }
+    }
+
+    //A12a
+    if (this.questions.get("Q17") && this.questions.get("Q18") && this.questions.get("Q19")) {
+      if(this.modes.get("A12a") !== undefined) {
+        this.modes.set("A12a", true);
+      }
+    }
+
+    //A12b
+    if (this.questions.get("Q17") && this.questions.get("Q18") && !this.questions.get("Q19")) {
+      if(this.modes.get("A12b") !== undefined) {
+        this.modes.set("A12b", true);
+      }
+    }
+
+    //A13
+    if (this.questions.get("Q14") && this.questions.get("Q15") && this.questions.get("Q16")) {
+      if(this.modes.get("A13") !== undefined) {
+        this.modes.set("A13", true);
+      }
+    }
+
+    //A14
+    if (this.questions.get("Q03") && this.questions.get("Q04") && this.questions.get("Q06")) {
+      if(this.modes.get("A14") !== undefined) {
+        this.modes.set("A14", true);
+      }
+    }
+
+    //A16
+    if (this.questions.get("Q07")) {
+      if(this.modes.get("A16") !== undefined) {
+        this.modes.set("A16", true);
+      }
+    }
+
+    //A18
+    if (this.questions.get("Q08")) {
+      if(this.modes.get("A18") !== undefined) {
+        this.modes.set("A18", true);
+      }
+    }
+
+    //A19
+    if (this.questions.get("Q29")) {
+      if(this.modes.get("A19") !== undefined) {
+        this.modes.set("A19", true);
+      }
+    }
+
+    //A21
+    if (this.questions.get("Q21")) {
+      if(this.modes.get("A21") !== undefined) {
+        this.modes.set("A21", true);
+      }
+    }
+
+    //A22
+    if (this.questions.get("Q09")) {
+      if(this.modes.get("A22") !== undefined) {
+        this.modes.set("A22", true);
+      }
+    }
+
+    //A23
+    if (this.questions.get("Q10")) {
+      if(this.modes.get("A23") !== undefined) {
+        this.modes.set("A23", true);
+      }
+    }
+
+    //A24
+    if (this.questions.get("Q11")) {
+      if(this.modes.get("A24") !== undefined) {
+        this.modes.set("A24", true);
+      }
+    }
+
+    //A25
+    if (this.questions.get("Q12")) {
+      if(this.modes.get("A25") !== undefined) {
+        this.modes.set("A25", true);
+      }
+    }
+
+    //A26
+    if (this.questions.get("Q13")) {
+      if(this.modes.get("A26") !== undefined) {
+        this.modes.set("A26", true);
+      }
+    }
+  }
+}
+
+export function highlightPath(clicks, butterflyPathsG, bodyColor = "orange", pathColor = "red") {
+
+  for (let i of clicks.modes) {
+
+    let color;
+    if (i[1]) {
```

Comment:
@Yuriko-Schumacher  This code is ugly, but it's getting the job done. We can change the colors once we have final colors from Naveen. We can also turn on citizenship text (once we have that in the SVG) in this function as well.